airframe-json/src/main/scala/wvlet/airframe/json/NullJSONContext.scala

```diff
@@ -24,12 +24,12 @@ class NullJSONContext(isObject: Boolean) extends JSONContext[Unit] with LogSuppo
   override def arrayContext(s: JSONSource, start: Int): JSONContext[Unit]  = new NullJSONContext(false)
   override def closeContext(s: JSONSource, end: Int): Unit                 = {}
 
-  override def add(v: Unit): Unit                                                                 = {}
-  override def singleContext(s: JSONSource, start: Int): JSONContext[Unit]                        = new NullJSONContext(false)
-  override def result: Unit                                                                       = {}
-  override def addNull(s: JSONSource, start: Int, end: Int): Unit                                 = {}
-  override def addString(s: JSONSource, start: Int, end: Int): Unit                               = {}
-  override def addUnescapedString(s: String): Unit                                                = {}
-  override def addNumber(s: JSONSource, start: Int, end: Int, dotIndex: Int, expIndex: Int): Unit = {}
-  override def addBoolean(s: JSONSource, v: Boolean, start: Int, end: Int): Unit                  = {}
+  override def add(v: Unit): Unit                                                = {}
+  override def singleContext(s: JSONSource, start: Int): JSONContext[Unit]       = new NullJSONContext(false)
+  override def result: Unit                                                      = {}
+  override def addNull(s: JSONSource, start: Int, end: Int): Unit                = {}
+  override def addString(s: JSONSource, start: Int, end: Int): Unit              = {}
+  override def addUnescapedString(s: String): Unit                               = {}
+  override def addNumber(str: String): Unit                                      = {}
```

Comment:
This interface change is OK, but if we can implement faster conversion from (JSONSource, start, end) -> number without using any String, the previous interface will be better.

Reply:
Thanks a lot. If we found a faster way, this change would be change and revert.